packages/flutter_tools/test/integration.shard/shader_compiler_test.dart

```diff
@@ -54,4 +74,66 @@ void main() {
     final int expectedMode = int.parse('644', radix: 8);
     expect(resultFile.statSync().mode & expectedMode, equals(expectedMode));
   });
+
+  testUsingContext('Compilation error with in storage', () async {
```

Comment:
Do these need a Context?

Reply:
Yeah, to construct the shader compiler and look up artifacts.